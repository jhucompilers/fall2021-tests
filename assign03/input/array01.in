PROGRAM arr;
  VAR months: ARRAY 12 OF INTEGER;

BEGIN
  months[0] := 31;
  months[1] := 28;
  months[2] := 31;
  months[3] := 30;

  WRITE months[3];
END.
