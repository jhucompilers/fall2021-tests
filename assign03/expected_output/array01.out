0,VAR,months,ARRAY 12 OF INTEGER
